api-docs/data-model.md

```diff
@@ -516,4 +516,54 @@ Unit of measurement: _volt-ampere reactive_.
 ### PayloadTemperature
 
  The temperature of the payload, e.g. in a cargo compartment.  
-Unit of measurement: _degrees Celcius_.
\ No newline at end of file
+Unit of measurement: _degrees Celcius_.
+
+### AcTotalRelativeActivePowerLoadPercentage
+
+ Total active power (W) consumption (of the connected load) as a percentage of the rated active power output capability.
+Unit of measurement: _percent_,
+
+### AcTotalRelativeApparentPowerLoadPercentage
+
+ Total apparent power (VA) consumption (of the connected load) as a percentage of the rated apparent output capability.
+Unit of measurement: _percent_.
+
+### AcTotalRelativeLoadPercentage
+
+ Total power load (taking both Actual (W) and Apparent (VA) consumption into consideration) as a percentage of the rated output capability.
+Unit of measurement: _percent_.
+
+### AcAveragePhaseToNeutralRmsVoltage
+
+ Average of the 3 phase-to-neutral voltages. This is an average of actual values and not an average over time.
+Unit of measurement: _volt_.
+
+### AcAveragePhaseToPhaseRmsVoltage
+
+ Average of the 3 phase-to-phase voltages. This is an average of actual values and not an average over time.
+Unit of measurement: _volt_.
+
+### AcAverageRmsCurrent
+
+ Average of the 3 individual phase (L1, L2, and L3) currents. This is an average of actual values and not an average over time.
+Unit of measurement: _ampere_.
+
+### Accelerometer_X_Axis
+
+ Acceleration (or g-force) measured on the x-axis of an accelerometer (typically placed inside a telematics device).
+Unit of measurement: _g-force_.
```

Comment:
'G' is the unit used by the device, not sure how it measure it, but your assumption is correct, 'G' means gravity acceleration and although it varies across the earth. 
The number you mention is the default conversion, but since we only have 'G' with 3 decimals I would not do a conversion with that precision (9.8 should do it).
That said, I am not sure anyone is interested in converting to m/s²

Reply:
OK, I will leave "g-force" as is.